contract/rust/Cargo.toml

```diff
@@ -9,7 +9,7 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
-serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
+serde_json = "*"
 near-bindgen = "0.6.0"
 borsh = "0.6.1"
```

Comment:
Should we make these other dependencies use `"*"` while we're here? Looks like we should be using `near-sdk` instead of `near-bindgen`; see other updates here https://github.com/near-examples/rust-status-message/pull/13#discussion_r416946300

Reply:
We would definitely benefit from a canonical "how do we like our dependencies". We are not consistent with this in examples right now - nft has serde_json = "1.0.45"
changed this one though

Reply:
Sorry, I was wondering if we should change borsh to `= "*"` while here, and otherwise make this file look like the one in my link above

That would also include an update from `near_bindgen = "0.6.0"` to `near_sdk = "0.10.0"`